src/main/java/com/graywolf336/trailingperspective/listeners/TrailingPerspectivePlayerListener.java

```diff
@@ -50,7 +51,11 @@ public void playerDoneDied(PlayerDeathEvent event) {
 
     @EventHandler
     public void playerTeleportedSomeWhere(PlayerTeleportEvent event) {
-        toggleTrailerTrailingPlayer(event.getPlayer());
+        TeleportCause cause = event.getCause();
+
+        if (cause.equals(TeleportCause.END_PORTAL) || cause.equals(TeleportCause.NETHER_PORTAL)) {
```

Comment:
Maybe change the `.equals` to `==`? Here's my reasoning, along with the fact this is how I usually do it: http://stackoverflow.com/a/1750453